index.js

```diff
@@ -1,7 +1,39 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Mariana";
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2 = "Aamir";
+console.log(`The navigator's name is ${hacker2}`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log("The driver has the longest name, it has 7 characters.");
+} 
+else {
+  console.log;
+}
 
 // Iteration 3: Loops
+
+const characters = hacker1.toUpperCase().split("").join(" ");
```

Comment:
Mariana, this is a correct solution for the exercise that we propose, however, you should've use a loop to reach the answer.
Here's another solution using a `for` loop:
```
for (let i = 0; i < hacker1.length; i++) {
  if (hacker1[i].localeCompare(hacker2[i]) === -1) {
    console.log("The driver's name goes first");
    break;
  } else if (hacker1[i].localeCompare(hacker2[i]) === 1) {
    console.log("Yo, the navigator's name goes first definitely");
    break;
  } else if (i === hacker1.length - 1) {
    console.log('What?! You both have the same name?');
  } else if (hacker1[i].localeCompare(hacker2[i]) === 0) {
    continue;
  }
}
```